Exercise05c.sh

```diff
@@ -0,0 +1,29 @@
+#creates a tab delimited file for easier pulling of data
```

Comment:
1.	1
2.	1
3.	.9– good code, more complex than it needs to be. All we’re looking for the what is the lowest wage listed for someone with 12 or 16 years experience respectively. You present the average wage for someone with 12 or 16 years experience. The difference between the minimums should be ~$4. Here’s some code that gets you most of the way there: cat wages.csv | cut -d , -f 3,4 | grep '16,' |  sort -t , -k 2 -n | head -n 1 | cut -d , -f 2 ; cat wages.csv | cut -d , -f 3,4 | grep '12,' |  sort -t , -k 2 -n | head -n 1 | cut -d , -f 2

2.9/3